app/controllers/episodes_controller.rb

```diff
@@ -1,12 +1,14 @@
 class EpisodesController < ApplicationController
   load_and_authorize_resource
-  before_action :set_episode, only: %i[edit show update]
+  before_action :set_episode, except: %i[index create episode_params new]
```

Comment:
episode_params? Is it a private method that can't be called directly via user?

Reply:
And I don't see `new` method here. So looks like you CP it from somewhere

Reply:
Removed episode_params from this scope.

I didn't CP. You don't see ```new``` method, but it is there though :)
https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-by-default-convention-over-configuration-in-action

But I'm not sure if callbacks work in such case..

Create podcast form works without empty ```new``` method, but I still add it for the sake of apparency.

Reply:
If you don't need new - remove it from routes, same for callback

Reply:
We DO need it for creating podcast and it works. So I keep it in callback exceptions and routes.